api/app/User.php

```diff
@@ -36,4 +37,19 @@ class User extends Authenticatable
     protected $casts = [
         'email_verified_at' => 'datetime',
     ];
+
+    public function jobForms()
+    {
+        return $this->hasMany(JobForm::class);
+    }
+
+    public function isJobFormOwner($id)
+    {
+        return in_array($id, $this->jobForms()->pluck('id')->all());
```

Comment:
Why was this needed? Why not just check auth'ed user id with a form's user_id? Unless this is returning if a user has forms filled, which I doubt.

Reply:
user can have many job forms. We get list of all his job forms. We check if given job form is in the list. Your option should be implemented in JobForm class. And i wanted to check if user is owner and if job form is owned by, because after admi nrole will be added you just can add || $this->isAdmin(); Ofc you can do similary in JobsForm. However, i did not implemnted filled forms owner and my idea was to put both checks under one method.  This will i look into a bit later.

Reply:
will fix latter if needed, when implementing other checks